src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -717,7 +717,7 @@ static void window_loadsave_paint(rct_window* w, rct_drawpixelinfo* dpi)
 
     // Draw path text
     auto ft = Formatter::Common();
-    ft.Add<uintptr_t>(Platform::StrDecompToPrecomp(buffer));
```

Comment:
The `Platform::StrDecompToPrecomp` is there for a reson, it seems, so instead of dropping it altogether, change its return type to be `utf8*` and stop doing the `reinterpret_cast` there

Then restore the usages of this function that you removed, please